### ISSUE TITLE
[Bug]: Ubuntu .deb(x64) App doesn't open after updating

### ISSUE THREAD

**Issue by Fruutebu** [FreeTubeApp/FreeTube#6696]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

After updating, app doesn't open anymore (clicking, right-clicking, neither on the app icon on desk nor on the app icon on app list). Tried and updated again but still doesn't work.

### Expected Behavior

Be able to open the app again

### Issue Labels

accessibility issue

### FreeTube Version

v.0.23.0 Beta

### Operating System Version

Ubuntu 18.04.6 LTS

### Installation Method

.deb

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Cpuld you go to the terminal and try starting FreeTube from there

**Comment by Fruutebu** [comment]
Thx for your answer.
Nope, on the terminal when i try to start : 
freetube: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by freetube)
freetube: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by freetube)
freetube: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /opt/FreeTube/libffmpeg.so)
freetube: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /opt/FreeTube/libffmpeg.so)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
The error messages you're seeing suggest that FreeTube is trying to use versions of the GNU C Library (GLIBC) that are not available on your system. Specifically, it requires GLIBC versions 2.28 and 2.29, but your system appears to have an older version.

GLIBC is an essential part of the Linux operating system, providing core functions like input/output, memory management, and system calls. Many applications depend on specific versions of GLIBC, and if your system doesn't have the required versions, the application will fail to run.

**Comment by Fruutebu** [comment]
Strange because freetube worked until today on my system. Bugged only after updating. I'll try to fix this if possible, thx for the answer

**Comment by absidue** [comment]
That is not surprising. FreeTube 0.22.x was based on Electron 32 which was released in August 2024 and FreeTube 0.23.x is based on Electron 34 which was released in January 2025. Considering that you are running an almost 7 year old major version of your distro, it was only a matter of time before you wouldn't be able to run modern software anymore. Normal support for Ubuntu 18 ended in 2023, Ubuntu Pro support for it ends in 2028, but only Ubuntu is promising to keep it alive if you pay them until then, everyone else is not going to wait that long.

If you want to run modern versions of FreeTube, as well Chromium and ffmpeg in general, make sure you are also using a modern version of your distro.

**Comment by leirbag28** [comment]
It is insane to destroy/update your entire OS just for an app when Appimage is supposed to be universal. I personally have tweaked the living hell out of my Ubuntu 18.04 with Professional apps. Ubuntu is updating versions too quickly. its insane. I am just getting started. I think at least the APPIMAGE should package the OLDER glibc in it as well as the flatpak..which is the poin.t of both of those systems. One of the Appimages I installed of a different app has Python 3.10 built into the Appimage.. solving the issue of me having to update my Python 3.8 hence ruining all the Python 3.8 related apps I installed. Please look into doing something like this for Appimage versions.

**Comment by absidue** [comment]
@leirbag28 If you are desperate to stick to an outdated version of a distro and use old software, then do so, you are welcome to use the older versions of FreeTube until they inevitably break when the world moves on from you. If you want to use modern software on stuff that is too old to run it, then it is on you to figure out a way to run it. We are a small group of volunteers that work on FreeTube in our spare time, we have much better things to do than stick to old versions of Electron, with known security issues, just so that a few people that refuse to keep up with the modern world can still run it. This is no different than the people that are still using Windows 7 and complain that no software wants to support it anymore.

**Comment by Fruutebu** [comment]
Even before your 1st answer, i had directly updated to Ubuntu 20.04.6 LTS and it worked again. Thx for answering and giving time over here. Cheers.

**Comment by leirbag28** [comment]
> [@leirbag28](https://github.com/leirbag28) If you are desperate to stick to an outdated version of a distro and use old software, then do so, you are welcome to use the older versions of FreeTube until they inevitably break when the world moves on from you. If you want to use modern software on stuff that is too old to run it, then it is on you to figure out a way to run it. We are a small group of volunteers that work on FreeTube in our spare time, we have much better things to do than stick to old versions of Electron, with known security issues, just so that a few people that refuse to keep up with the modern world can still run it. This is no different than the people that are still using Windows 7 and complain that no software wants to support it anymore.

Well Thank you for all that All of you do. I guess I will stick to older versions. WHile I do HIGHLY disagree in every way imaginable that NEWER OSes are better and more secure, I definitely do not follow the Pattern of the world (which is one of the reason I am on Linux). Ubuntu 18.04 works...the newer versions don't for many reasons.... Wayland is NOT for professional work yet nor is Pipewaire. it is as simple as That. I use my Linux for VJ-ing and DJ-ing and Karaoke and Video Mapping. Can't afford to break it. Thats all.
